### PR TITLE
[test] check files in auto-update config file map

### PR DESCRIPTION
**Pull request by x09326**
PR for check empty files field in auto-update config.
And I have checked all **package.json**.
There is no affected lib.

cc @PeterDaveHello , cc #9299

### CODE REVIEW

File: test/valid-packages-test.js [hunk 1]
```diff
@@ -307,6 +307,19 @@ packages.map(function(pkg) {
         }
       }
   };
+  packageVows[pname + ": There must be array type files in auto-update config"] = function(pkg) {
+    var json = parse(pkg, true);
+    if (json.npmFileMap) {
+      for (var i in json.npmFileMap) {
+        assert.ok(Array.isArray(json.npmFileMap[i].files),
+                  pkgName(pkg) + ": Need to add Array type files in auto-update config");
```
Comment (PeterDaveHello) [inline test/valid-packages-test.js]: Try something like "files in auto-update config file map need to be an array"

**Comment by PeterDaveHello** [comment]
@pvnr0082t please also join us!

File: test/valid-packages-test.js (hunk 1 above)
Comment (x09326) [inline test/valid-packages-test.js]: I have modified it.
@PeterDaveHello @pvnr0082t 
Please help me review it.

**Comment by pvnr0082t** [comment]
@x09326 Please fix commit message as PR title. Thanks.

**Comment by x09326** [comment]
@pvnr0082t @PeterDaveHello 
I have modified the commit message.
Please help me review it, thanks.

**Comment by x09326** [comment]
ping @pvnr0082t @PeterDaveHello 
I have rebased to the latest branch.
Please help me review it.

File: test/valid-packages-test.js [hunk 2]
```diff
@@ -307,6 +307,19 @@ packages.map(function(pkg) {
         }
       }
   };
+  packageVows[pname + ": There must be array type files in auto-update config"] = function(pkg) {
```
Comment (pvnr0082t) [inline test/valid-packages-test.js]: @x09326 Array is not a type. I think `array datetype` will be better.

Comment (x09326) [inline test/valid-packages-test.js]: datetype or datatype?
I think it is datatype.

Comment (x09326) [inline test/valid-packages-test.js]: @pvnr0082t 
I have modified it.

**Pull request merged**